.travis.yml

```diff
@@ -1,6 +1,9 @@
 dist: xenial
 language: python
 python: 3.7
+addons:
+  hosts:
+    - pulp
```

Comment:
What is this meant for?
Is there an addon service in travis that provides a pulp server?

Reply:
Found it. ;)
https://docs.travis-ci.com/user/hosts/

Reply:
I wish! This is [to set a custom hostname in travis](https://docs.travis-ci.com/user/hosts/).

Reply:
You beat me. I should probably use a different name to avoid confusion. `pulp-fixtures` maybe?

Reply:
Yes, good point.